certification/internal/shell/has_license.go

```diff
@@ -8,7 +8,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type HasLicenseCheck struct{}
+type HasLicenseCheck struct {
+	certification.DefaultCheck
```

Comment:
certification.DefaultCheck.  I'd like to discuss this.  I'm not sure there are default checks.  I think there are checks for certain Certifications.  Container Cert = a set of checks, Operator Cert = a set of checks.

Reply:
It is not meant to be considered a default other than as a default implementation of the isBundleCheck and isBundleCompatibile, so that it didn't need to overridden in every check. Only those checks that need to can override to provide a value different from the default.

The other option to this implementation is just to have a different list (as you implied) and just change which list is run based on if it is a bundle or not.